jhrasco-ct/30-Days-Of-React
# RCT-150: 30 Days of React - Day 1

## Related Tickets

- [RCT-150: React Training](https://codeandtheory.atlassian.net/browse/RCT-150)

## Description

Completed the exercises in the [30 Days of React - Day 1](https://github.com/jhrasco-ct/30-Days-Of-React/blob/master/01_Day_JavaScript_Refresher/01_javascript_refresher.md).

## Validation

1. Open each JavaScript file containing the code for each exercise.
2. Verify that the solutions written are correct and satisfy the challenges.
3. To verify, open the **index.html** file and inspect the output using the browser's console.
